### PR TITLE
* Add conn ID to the transaction log

### PR DESCRIPTION
**Pull request by zimulala**
If a transaction occurs an error, we need to begin a new transaction to retry. Then we don't know the relation between these transactions in the original log. So I add the conn ID to transaction log to fix it.

### CODE REVIEW

File: store/tikv/2pc.go [hunk 1]
```diff
@@ -79,10 +79,11 @@ type twoPhaseCommitter struct {
 	}
 	priority pb.CommandPri
 	syncLog  bool
+	connID   uint64 // connID is used for log.
```
[the remark this answers is not in the document]
Comment (zimulala) [reply store/tikv/2pc.go]: If you do as you say, each twoPhaseCommiter‘s function needs to get the value from `context.Context`. I think there will be more overhead in that case.

**Comment by tiancaiamao** [comment]
LGTM
/run-all-tests

**Comment by zimulala** [comment]
PTAL @coocood @disksing

**Comment by coocood** [comment]
LGTM

**Comment by disksing** [comment]
LGTM.

**Pull request merged**